doc/source/release-notes/Percona-Server-8.0.25-15.1.rst

```diff
@@ -0,0 +1,47 @@
+.. _8.0.25-15.1:
+
+================================================================================
+*Percona Server for MySQL* 8.0.25-15.1
+================================================================================
+
+:Date: June 16, 2021
+:Installation: `Installing Percona Server for MySQL <https://www.percona.com/doc/percona-server/8.0/installation.html>`_
+
+`Percona Server for MySQL <https://www.percona.com/software/mysql-database/percona-server>`_ 8.0.25-15.1
+includes all the features and bug fixes available in
+`MySQL 8.0.25 Community Edition <https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-25.html>`_
+in addition to enterprise-grade features developed by Percona.
+
+The TokuDB Storage Engine will be disabled in future Percona Server for MySQL 8.0 versions. We recommend migrating to the MyRocks Storage Engine. 
```

Comment:
Please include a pointer to the blog post that provides additional details.
See [PS-7426](https://jira.percona.com/browse/PS-7426) for details.

Reply:
Also, can we make this paragraph stand out using a note box?

Reply:
Added the link to the blog post. 
Note added.